packages/server/cron.yaml

```diff
@@ -0,0 +1,4 @@
+cron:
+  - description: "test"
```

Comment:
nit: add actual description? seems like something that can be forgotten easily :P